decNumber/include/decNumber/decDouble.h

```diff
@@ -38,7 +38,7 @@
   #define DECDOUBLE_Ehigh (DECDOUBLE_Emax + DECDOUBLE_Bias - (DECDOUBLE_Pmax-1))
 
   /* Required includes                                                */
-  #include "decContext.h"
+  #include "include/decNumber/decContext.h"

```

Comment:
any reason not to use the following form here?

`#include <decNumber/decContext.h>`

Reply:
ditto